.travis.yml

```diff
@@ -46,13 +44,18 @@ install:
       (cd src && make SYSCFLAGS="-DLUA_USE_LINUX -ULUA_COMPAT_5_2" SYSLIBS="-Wl,-E -ldl -lreadline" LUA_A=liblua.so MYCFLAGS="-fPIC" RANLIB=: AR="gcc -shared -ldl -o" liblua.so) || exit 1
       sudo make INSTALL_TOP=/usr/ INSTALL_INC=${LUAINCLUDE} TO_LIB=liblua.so linux install || exit 1
       cd ..
+    elif [[ "$LUANAME" == "luajit-2.0" ]]; then
```

Comment:
Could you move this into its own block (`if`) instead of using `elif`?`That way this will be easier to find when Travis folds the commands.